### PR TITLE
allow governance body on parachain to have sovereign account

### PR DESCRIPTION
**Pull request by xlc**
The goal is to allow Fellowship on Collective chain to have a sovereign account on Polkadot so that we can add it as an identity registrar. This will allow Fellows origin to be able to provide judgements for Fellowship members.

This currently allow any body on any parachain including non system parachains to have sovereign account. I cannot think of any reason why that may be an issue but let me know if I should change it to filter only system parachains.

[This](https://gist.github.com/xlc/ec61bfa4e9f6d62da27d30141ad2c72b) is the testing script.

Original PR: https://github.com/paritytech/polkadot/pull/7518

### CODE REVIEW

**Review by KiChjang — APPROVED** [review]

**Comment by xlc** [comment]
Anyone want to hit the merge button? It won't kill you.

**Comment by KiChjang** [comment]
This still requires an audit before we can merge/release.

**Comment by xlc** [comment]
> This still requires an audit before we can merge/release.

ok. that’s fine. is there a label to indicate that?

**Comment by KiChjang** [comment]
Not that I know of; someone from devops should look into this, perhaps @the-right-joyce?

**Comment by the-right-joyce** [comment]
labels are not any longer used for the auditing process. Instead PRs should be added to the [security audit board](https://github.com/orgs/paritytech/projects/103/views/1) as mentioned [here](https://github.com/paritytech/polkadot-sdk/blob/master/docs/CONTRIBUTING.md#process)

I see that your PR was archived though, will check with SRLabs and come back to you

**Comment by the-right-joyce** [comment]
@KiChjang it was already audited, we can merge

File: polkadot/runtime/polkadot/src/xcm_config.rs
Comment (ggwpez) [inline polkadot/runtime/polkadot/src/xcm_config.rs:72]: @mustermeiszer here

**Pull request merged**
